### PR TITLE
Update broken links to commercetools docs

### PR DESCRIPTION
**Pull request by industrian**
#### Summary/Description

This PR updates the links to commercetools docs pages, as they were all broken.

#### Todo

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [ ] Documentation
- [ ] `Type` label for the PR

### CODE REVIEW

File: packages/price-exporter/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # price-exporter
 
-Export [commercetools price](https://dev.commercetools.com/http-api-projects-products.html#price) from the commercetools platform.
+Export [commercetools price](https://docs.commercetools.com/api/projects/products#price) from the commercetools platform.
```
Comment (industrian) [inline packages/price-exporter/readme.md:3]:
```suggestion
Export [commercetools price](https://docs.commercetools.com/api/types#price) from the commercetools platform.
```

File: packages/csv-parser-price/readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # commercetools-csv-parser-price
 
-Convert [commercetools price](https://dev.commercetools.com/http-api-projects-products.html#price) CSV data to JSON
+Convert [commercetools price](https://docs.commercetools.com/api/projects/products#price) CSV data to JSON
```
Comment (industrian) [inline packages/csv-parser-price/readme.md:3]:
```suggestion
Convert [commercetools price](https://docs.commercetools.com/api/types#price) CSV data to JSON
```

File: packages/resource-deleter/readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # resource-deleter
 
-Deletes [commercetools resources](https://docs.commercetools.com/http-api.html#reference-expansion) from the commercetools platform.
+Deletes [commercetools resources](https://docs.commercetools.com/api/general-concepts#reference-expansion) from the commercetools platform.
```
Comment (industrian) [inline packages/resource-deleter/readme.md:3]:
```suggestion
Deletes commercetools resources from the commercetools platform.
```

File: docs/sdk/api/sdkAuth.md [hunk 4]
```diff
@@ -229,7 +229,7 @@ await authClient.customFlow({
 
 ### Token Introspection
 
-Fetches info about `access_token` using [Token Introspection](https://docs.commercetools.com/http-api-authorization.html#oauth2-token-introspection) from the commercetools platform API.
+Fetches info about `access_token` using [Token Introspection](https://docs.commercetools.com/api/authorization#oauth2-token-introspection) from the commercetools platform API.
```
Comment (industrian) [inline docs/sdk/api/sdkAuth.md:232]:
```suggestion
Fetches info about `access_token` using [Token Introspection](https://docs.commercetools.com/api/authorization#introspection) from the commercetools platform API.
```

**Review by markus-azer — APPROVED** [review]
Thanks @industrian 💯 
is there any quick automation tool that could check these links same as the one used in doc?!

**Pull request merged**